packages/chat-stateful-client/src/ChatContext.ts

```diff
@@ -317,6 +325,54 @@ export class ChatContext {
     }
   }
 
+  /**
+   * Tees any errors encountered in an async function to the state.
```

Comment:
Is `tee` something easily understandable to non-english first language devs or is it a colloquial verb? (another option could be propagate but I do like tee)

Reply:
`tee` is standard for linux developers: https://www.man7.org/linux/man-pages/man1/tee.1.html

Seems to mean the same thing on most platforms: https://en.wikipedia.org/wiki/Tee_(command)